numpy/__init__.pyi

```diff
@@ -522,15 +522,29 @@ def array(
     copy: bool = ...,
     subok: bool = ...,
     ndmin: int = ...,
+    *,
```

Comment:
Fyi, an `*` was recently added to `array()` in https://github.com/numpy/numpy/pull/16966.
I'm not 100% sure what will happen if we merge now, but I suspect we'll end up with two asterisks (which is bad).

Reply:
I wonder who was responsible for adding that `*`! 😄 

Thanks for the heads up, I merged master and fixed that in https://github.com/numpy/numpy/pull/16935/commits/fadf47334279bf63c59c61d309462a8ed68c7cb7 .